yb-voyager/src/tgtdb/oracle.go

```diff
@@ -567,7 +567,7 @@ func (tdb *TargetOracleDB) ExecuteBatch(migrationUUID uuid.UUID, batch *EventBat
 
 func (tdb *TargetOracleDB) InitConnPool() error {
 	if tdb.tconf.Parallelism == -1 {
-		tdb.tconf.Parallelism = 1
+		tdb.tconf.Parallelism = 16
```

Comment:
Can you ensure that the fall-back setup/fall-forward setup commands have the appropriate CLI help message for parallel-jobs?

It should indicate the default for oracle (i.e. 16)

Reply:
Sure